### ISSUE TITLE
[Bug]: rstcheck confuses `include` directive with `code-block`

### ISSUE THREAD

**Issue by ReenigneArcher** [rstcheck/rstcheck#184]
### To Dos

- [X] I tested with the [latest released version](https://github.com/rstcheck/rstcheck/releases/latest).

- [X] I have checked the [issues](https://github.com/rstcheck/rstcheck/issues) and think that this is not a duplicate.

- [X] I added a very descriptive title to this issue.

### Example Code (rst)

```rst
Table of Contents
=================
.. include:: toc.rst
```

### Relevant log output

```Shell
CRITICAL:rstcheck_core.checker:An `AttributeError` error occured. This is most propably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'docs/source/index.rst'. See https://rstcheck-core.readthedocs.io/en/latest/faq/#code-blocks-without-language-sphinx for more information.

CRITICAL:rstcheck_core.checker:An `AttributeError` error occured. This is most propably due to a code block directive (code/code-block/sourcecode) without a specified language. This may result in a false negative for source: 'docs/source/about/overview.rst'. See https://rstcheck-core.readthedocs.io/en/latest/faq/#code-blocks-without-language-sphinx for more information.
```

### Description

I am not sure if this is a false positive or what, but I don't have any code-blocks without a specified language.

The first line in the output points to the file pasted above... A pretty simple file. I don't know if the error is actually in that file or within what's included, which is here:

```rst
.. toctree::
   :maxdepth: 2
   :caption: About

   about/overview
   about/installation
   about/docker
   about/third_party_packages
   about/usage
   about/app_examples
   about/advanced_usage
   about/changelog

.. toctree::
   :maxdepth: 2
   :caption: GameStream

   gamestream/gamestream

.. toctree::
   :maxdepth: 2
   :caption: Troubleshooting

   troubleshooting/general
   troubleshooting/linux
   troubleshooting/macos
   troubleshooting/windows

.. toctree::
   :maxdepth: 2
   :caption: Build

   building/build
   building/linux
   building/macos
   building/windows

.. toctree::
   :maxdepth: 2
   :caption: Contributing

   contributing/contributing
   contributing/localization
   contributing/testing

.. toctree::
   :maxdepth: 2
   :caption: Legal

   legal/legal

.. toctree::
   :maxdepth: 2
   :caption: source

   source/src
```

The second error points to this file, which also contains an include directive, and in this case nothing else.

```rst
.. include:: ../../../README.rst
```

There are no code blocks in the included file.

Perhaps this library doesn't understand the `include` directive?

### Operating System

Linux

### Operating System Details

This is running in a readthedocs build, Ubuntu 22.04.

### Python Version

3.11

### rstcheck Version

6.1.2

**Comment by ReenigneArcher** [comment]
Resolving the c++ code issue causing the error in #186, resolved these two errors. In any event, the two errors printed do not seem relevant to the actual issue (a c++ compile error).
